hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -164,6 +164,11 @@ def execute_as!(user, command, do_say = true, do_log = true, extra_env = {})
     execute!(runuser_command, do_say, do_log, extra_env)
   end
 
+  def execute_as_formatted!(user, command, do_say = true, do_log = true, extra_env = {})
+    runuser_command = ['runuser', '-l', user, '-c', command]
+    execute_command(runuser_command, do_say, do_log, extra_env)
```

Comment:
I wonder if `do_say` and `do_log` can deal with an array properly.

Reply:
```
2024-12-03 12:38:14 [DEBUG ] [pre] Updating ownership of the evr extension
2024-12-03 12:38:14 [DEBUG ] [root] Executing: ["runuser", "-l", "postgres", "-c", "psql -d foreman -c \"UPDATE pg_extension SET extowner = (SELECT oid FROM pg_authid WHERE rolname='foreman') WHERE extname='evr'\""]
2024-12-03 12:38:14 [DEBUG ] [root] UPDATE 1
```

This is how it's logged..

Reply:
I'll update logging to join the array and log so it's displayed properly.